deno/fhir-embeddings/store.ts

```diff
@@ -0,0 +1,76 @@
+import * as fs from "fs";
+import "https://deno.land/std@0.192.0/dotenv/load.ts";
+import {
+  createClient,
+  SupabaseClient,
+} from "https://esm.sh/@supabase/supabase-js@2";
+
+import { Document } from "https://esm.sh/langchain/document";
+import { OpenAIEmbeddings } from "https://esm.sh/langchain/embeddings/openai";
+import { SupabaseVectorStore } from "https://esm.sh/langchain/vectorstores/supabase";
+
+function compileSearchParamsDocuments(searchParamsSpec: any): Document[] {
```

Comment:
Do we know the type of the params?

Reply:
we are owning the ingested data in `data/fhir-r4b-search-parameters.json` and this is the only place where we get to use it.  So I'll skip typing this _deep JSON_ for now.